### ISSUE TITLE
Mask Normalizing in FL_InpaintCrop.py

### ISSUE THREAD

**Issue by IrradiatedHaggis** [filliptm/ComfyUI_Fill-Nodes#107]
In FL_InpaintCrop.py the current squeeze function can cause some issues if you feed it some types of masks.

I changed this:
```python
        if mask.shape[1] != image.shape[1] or mask.shape[2] != image.shape[2]:
            mask = F.interpolate(mask.unsqueeze(1), size=(image.shape[1], image.shape[2]), mode='nearest').squeeze(1)
            print(f"[FL Inpaint Crop] Auto-resized mask to {image.shape[2]}x{image.shape[1]}")
``` 

To this:
```python
        # Auto-resize mask to match image size
        if mask.shape[-2] != image.shape[1] or mask.shape[-1] != image.shape[2]:
        
            # Normalize mask to NCHW
            if mask.dim() == 2:
                mask_nchw = mask.unsqueeze(0).unsqueeze(0)
            elif mask.dim() == 3:
                if mask.shape[0] == 1:
                    mask_nchw = mask.unsqueeze(1)
                else:
                    mask_nchw = mask.unsqueeze(0)
            elif mask.dim() == 4:
                mask_nchw = mask
            else:
                raise ValueError(f"Unsupported mask shape: {mask.shape}")

            # 🔥 nchw to float
            mask_nchw = mask_nchw.float()

            # Resize safely
            mask_resized = F.interpolate(
                mask_nchw,
                size=(image.shape[1], image.shape[2]),
                mode='nearest'
            )
        
            # Return to [1, H, W]
            mask = mask_resized.squeeze(0).squeeze(0)
        
            # Clamp to valid mask range
            mask = torch.clamp(mask, 0.0, 1.0)
        
            print(f"[FL Inpaint Crop] Auto-resized mask to {image.shape[2]}x{image.shape[1]}")
```

**Comment by filliptm** [comment]
i can implement this fix but if youd like your welcome to put in a PR to get contributor status on the repo

**Comment by filliptm** [comment]
Fixed by PR #108 plus follow-up commit 3078ffa. FL_InpaintCrop now normalizes 2D/3D/4D masks to [B,H,W], safely resizes masks to the image dimensions, clamps mask values, and applies the same handling to optional context masks.
